UPGRADE.md

```diff
@@ -1,6 +1,6 @@
 # Upgrade
 
-## dev-develop
+## 2.0.0-alpha5
```

Comment:
That line feels odd, because this change should actually not be there... I've got a bad feeling, because somebody messed with the repository last week :see_no_evil: Can you try to rebase?